package com.tmt.kontroll.test.persistence.run.utils.exceptions;

@SuppressWarnings("serial")
public class TestPreparationFailedException extends RuntimeException {

	public TestPreparationFailedException(final Throwable cause) {
		super(cause);
	}
}
